Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
+    
+    func getKey(forString: String) -> [String] {
+        
+        guard
+            let value = parseResponse(response: forString),
+            let filePath = applicationStateHandler.filePath?.appendingPathComponent("build/Calabash.app/Frameworks/XNGLocalizedString.framework/en.lproj/Localizable.strings"),
+            let stringsDict = NSDictionary(contentsOf: filePath) as? [String: String]
+        else { return [] }
+        
+        return stringsDict.keysForValue(value: value)
+    }
+    
+    private func parseResponse(response: String) -> String? {
```

Comment:
Nit, especially since the function is private anyway, but maybe rename this to `parseResponse(_ response)` to avoid duplication.